wurst/objects/abilities/priest/unsub/MixSpells.wurst

```diff
@@ -91,6 +106,8 @@ function mixMana(unit caster, unit target)
     let casterMana = caster.getMana()
     let newMana = (targetMana + casterMana) / 2.00
 
+    createFadingTextTag(target, getMixDiffStr((targetMana - newMana).toInt()), COLOR_TURQUOISE)
+    createFadingTextTag(caster, getMixDiffStr((casterMana - newMana).toInt()), COLOR_TURQUOISE)
```

Comment:
Why not use `COLOR_ENERGY` or what is used for Mana Burn?

Reply:
I used turquoise as it brighter, thought it would be easier to notice